docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/checkbox/checkbox.twig

```diff
@@ -5,9 +5,11 @@
  *
  * Variables:
  * - theme: [string] Theme: light, dark.
+ * - text: [string] Input field label.
```

Comment:
could you please update `text` to `content` - sorry, I wrote incorrectly in the ticket - fixed now